packages/generators/app/lib/resources/json/ts/tsconfig-server.json.js

```diff
@@ -22,8 +22,6 @@ module.exports = () => ({
     '.cache/',
     '.tmp/',
 
-    // Do not include admin files in the server compilation
-    'src/admin/',
```

Comment:
Since this is the server tsconfig file, I don't think we should remove the `src/admin` directory from the excluded property here.